var/spack/repos/builtin/packages/madgraph5amc/package.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+
+class Madgraph5amc(Package):
+    """MadGraph5_aMC@NLO is a framework that aims at providing
+       all the elements necessary for SM and BSM phenomenology,
+       such as the computations of cross sections, the generation
+       of hard events and their matching with event generators,
+       and the use of a variety of tools relevant to
+       event manipulation and analysis. """
+
+    homepage = "https://launchpad.net/mg5amcnlo"
+    url      = "https://launchpad.net/mg5amcnlo/2.0/2.7.x/+download/MG5_aMC_v2.7.3.tar.gz"
+
+    version('2.7.3', sha256='0b665356f4d9359e6e382e0f408dc11db594734567c6b2f0ec0e0697f2dbe099', preferred=True)
+    version('2.7.3.py3', sha256='400c26f9b15b07baaad9bd62091ceea785c2d3a59618fdc27cad213816bc7225')
```

Comment:
Since Python 2 is EOL, would it be better to default to the Python 3 version instead?